### DISCUSSION TITLE
Simple PWA example for idea generation

### DISCUSSION THREAD

**Discussion by ap1969** [quasarframework/quasar#9761] [Show and Tell]
Hi all,

I recently launched a PWA using Quasar, at https://app.yourideamachine.com - it's a quick-and-dirty tool to help people that are creating their own products/services online, to either generate new ideas or flesh out an hold one. The database still needs a lot of fleshing out in terms of it's content, but it's up and running.

Backend is feathersjs, authentication via Auth0, if anyone is interested.

Regards,
Andy

**Comment by ontwikkelfabriek** [comment]
@ap1969 

> Backend is feathersjs, authentication via Auth0, if anyone is interested.

Yes very interested , that combo would be an excellent stack for a Quasar starter kit with backend and auth integration?


About the site: Very nice.  I am going to take a look!
